ui/src/components/Preview/index.jsx

```diff
@@ -1,45 +1,78 @@
+import classNames from 'classnames';
```

Comment:
If you do:

```jsx
yarn styleguide
```

this would open the styleguide for this site. components in `ui/components` that have `README.md` will be displayed in the styleguide. For example, you can look at the StatusLabel component https://github.com/taskcluster/taskcluster/blob/master/ui/src/components/StatusLabel/README.md.

We should have one for this component. Under the hood, we are using react-styleguide to do this. Feel free to look at https://react-styleguidist.js.org/docs/documenting.html#writing-code-examples.